style.css

```diff
@@ -269,6 +269,20 @@ li {
   margin-bottom: 10px;
 }
 
+.popupbottombtn button {
+  display: block;
+  margin: auto;
```

Comment:
- [ ] And I think because you have duplicated code, you might have some in your CSS to, please inspect your file and adjust everything, in my case, I will hilght some on which you need to work on before your project gets approved, 
- Please consider removing the black shallow on the technology button, take a look at this original image on the left and apply the changes.
- The second is the text circled in red, make sure you use the correct font and alignment as per the original on the left
- last but not least, make your web app breath a bit by adding more padding where shown compared to the original on the left
- GOOD LUCK

<img width="603" alt="1" src="https://user-images.githubusercontent.com/77843050/209299288-163447cd-f1a4-47de-9a94-69505d4de5a6.png">

<img width="771" alt="2" src="https://user-images.githubusercontent.com/77843050/209299297-c01b7d16-2216-488d-aea2-9c22575d4dc1.png">